src/main/java/duke/Event.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Event extends Task {
+
+    protected String at;
+
+    public Event(String description, String at, boolean isDone) {
+        super(description, isDone);
+        String date;
+        try {
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+            date = LocalDate.parse(at, formatter).format(DateTimeFormatter.ofPattern("MMM dd yyyy"));
+        } catch (DateTimeParseException e) {
+            date = at;
+        }
+
+        this.at = date;
+    }
+    public Event(String description, String at) {
```

Comment:
```suggestion
    }
    
    public Event(String description, String at) {
```
Perhaps you could add a line break to improve readability?